### DISCUSSION TITLE
Got blocked by the firewall, when using xtls proxy to download mp4 video files

### DISCUSSION THREAD

**Discussion by CelloDude** [XTLS/Xray-core#588] [Q&A]
Recently, when using xtls proxy to download mp4 video files from catchvideo.net and convertirvideo.com, the proxy traffic got blocked by the firewall straightaway. I know quite little about computer networks and coding, so sorry if this was a stupid question:
Despite being encrypted by tls, could the "downloading mp4" type of tls traffic be easily recognized and blocked by the firewall?

Many thanks in advance, your help and explanations are very much appreciated.

PS: A bit of a wild guess: that the upgraded and deep-learning trained firewall might be able identify some specific types of "not normal enough" tls-encrypted traffic.

**Comment by RPRX** [comment]
> Recently

Please let me know if you can reproduce it **when not that period of time.**

**Reply by CelloDude** [reply]
Sure.
By the way, haven't tested recently, but I used to run other proxy protocols (e.g. wg) through the xtls proxy tunnel.
Then, found that the firewall was able to block some of them (especially wg) after a while.
After realizing that it was probably not good practice to run firewall-vulnerable proxies through firewall-resistant ones, I stopped trying that.

Thanks so much for all the time and efforts invested into developing this proxy.
You're truely a network programing rock star hero to me!🚀🌟👨‍💻
